sdk/core/System.ClientModel/src/Internal/BitVector640.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.ClientModel.Internal;
+
+internal struct BitVector640
```

Comment:
could we add a description of what this type does?

Reply:
Yep, happy to!

Reply:
Addressed